cmd/tendermint/commands/light.go

```diff
@@ -64,6 +67,8 @@ var (
 	dir                string
 	maxOpenConnections int
 
+	daSampling     bool
+	numSamples     int32
```

Comment:
This will never be negative, any reason to not use `uint32` here directly (and `Uint32Var(...)`) instead of doing an unsafe cast later?